gateway-protocol/src/main/proto/gateway.proto

```diff
@@ -549,6 +549,16 @@ message ModifyProcessInstanceResponse {
 
 }
 
+message DeleteResourceRequest {
+  // The key of the resource that should be deleted. This can either be the key
+  // of a process definition, or the key of a decision requirements definition.
```

Comment:
```suggestion
  // of a decision requirements definition.
```

❓ For now, only DRG deletion is supported, right? Should we just expand this documentation later, when process deletion is added? Or would that be unnecessary effort, as we'll implement it before 8.2 anyways? I guess so, but just wanted to raise it. 🤷

Reply:
I was struggling with this same question when I wrote it. I feel like we should just keep it in here as we should've implemented it before 8.2 so we don't have to revisit it. For an alpha version it really doesn't matter if the documentation is not 100% accurate.